package.json

```
@@ -5,10 +5,12 @@
     "main": "dist/framer-motion.cjs.js",
     "module": "dist/es/index.js",
     "exports": {
-        "package.json": "./package.json",
-        "require": "./dist/framer-motion.cjs.js",
-        "import": "./dist/es/index.js",
-        "default": "./dist/es/index.js"
+        ".": {
+            "require": "./dist/framer-motion.cjs.js",
+            "import": "./dist/es/index.js",
+            "default": "./dist/es/index.js"
+        },
+        "package.json": "./package.json"
```

This needs to start with `./`
```suggestion
        "./package.json": "./package.json"
```